app/models/product.rb

```diff
@@ -73,6 +73,7 @@ def self.billing_modes
   scope :active, -> { where(is_archived: false, is_hidden: false) }
   scope :alphabetized, -> { order(Arel.sql("LOWER(products.name)")) }
   scope :archived, -> { where(is_archived: true) }
+  scope :available_for_estimates, -> { where({ type: %w[Item Service Instrument TimedService Bundle] }).not_archived }
```

Comment:
should we use `active` instead of `not_archived`? I'm not sure what the hidden flag means though

Reply:
Hidden products are the ones that the users can't purchase by themselves but can be ordered through "Order for". I think between cores it's fine to order hidden products.